packages/@n8n/task-runner-python/src/health.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
```

Comment:
I'm a bit surprised we're doing this with `asyncio` instead of `http` (and threading), but I guess the use case really is that simple that asyncio does the job 🤔

And all things considered this is very clean & minimal compared to some healthchecks I've implemented in the past, so that's a 👍 from me after the initial shock.